src/examples/server/ssePollingExample.ts

```diff
@@ -112,6 +109,9 @@ app.use(cors());
 // Create event store for resumability
 const eventStore = new InMemoryEventStore();
 
+// Track transports by session ID for session reuse
+const transports = new Map<string, StreamableHTTPServerTransport>();
+
```

Comment:
```suggestion
```

don't think you need this any more

Reply:
I thought so too at first, but we do still need this at line 120 below:

```
// Reuse existing transport or create new one
let transport = sessionId ? transports.get(sessionId) : undefined;
```